packages/front-end/components/Experiment/CompactResults.tsx

```diff
@@ -24,8 +26,22 @@ const percentFormatter = new Intl.NumberFormat(undefined, {
   maximumFractionDigits: 2,
 });
 
-function hasEnoughData(value1: number, value2: number): boolean {
-  return Math.max(value1, value2) >= 150 && Math.min(value1, value2) >= 25;
+function hasEnoughData(
+  value1: number,
+  value2: number,
+  sigThreshold: number = defaultMinConversionThresholdSignificance
+): boolean {
+  return Math.max(value1, value2) >= sigThreshold;
+}
+
+function hasVarianceWarning(
+  value1: number,
+  value2: number,
+  varianceThreshold: number = defaultVarianceThreshold
+): boolean {
+  return (
+    Math.abs(value1 - value2) / Math.max(value1, value2) >= varianceThreshold
```

Comment:
I'm thinking we may want to always use the control as the denominator instead of the max.  I think that makes it easier to explain and reason about.  Setting a limit of 50% means if a variation is up more than 50% or down more than 50% it warns you.

Currently, if the control is 100 and the variation is 150, it would calculate the difference as 0.33 when it should be treated as 0.5.